owners/src/parser.js

```diff
@@ -37,23 +46,29 @@ class OwnersParser {
    * Parse an OWNERS.yaml file.
    *
    * @param {!string} ownersPath OWNERS.yaml file path.
-   * @return {OwnersRule} parsed OWNERS file rule.
+   * @return {OwnersParserResult} parsed OWNERS file rule.
```

Comment:
add `!`

Reply:
`!` is not used in the `@return` tag, only in `@param` tags. `@return` requires an explict `null|Type` if the return value is optional/nullable.

Reply:
Update: Not sure where I read this, but I distinctly recall using `!` until I was corrected/got some lint error/typescript warning to remove them. Now I don't know what to believe. At some point I'll be updating eslint's config around JSDoc (specifically `Array<T>` vs `T[]`; I'll look into this then.